spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/Binder.java

```diff
@@ -32,21 +32,21 @@
 public interface Binder<T> {
 
 	/**
-	 * Bind a message consumer on a channel
+	 * Bind the target component as a message consumer to the logical entity identified by the name.
```

Comment:
I thought about it when adding this and convinced myself with the explanation at the class level what the logical entity could be. do you think that isn't enough?